frontend/packages/kubevirt-plugin/src/actions/provider.ts

```diff
@@ -83,3 +87,17 @@ export const useVmiActionsProvider = (vm: VMKind) => {
   }, [k8sModel, vm, vmStatusBundle, vmi]);
   return React.useMemo(() => [actions, !inFlight, undefined], [actions, inFlight]);
 };
+
+export const useModifyApplicationActionProvider = (element: GraphElement) => {
```

Comment:
Is this adding an action specific to virtual machines or this action is added to other resources as well?

Reply:
this is specific to virtual machines

Reply:
> we add this action only if the node type is a workload and in this case, it is a virtual machine, 

hi, what is missing in a virtualmachines resource to identify it as workloads?

Reply:
@yaacov Only these resources 'deployments', 'deploymentConfigs', 'daemonSets', 'statefulSets', 'jobs', 'cronJobs', 'pods' are identified as workload types.

@rohitkrai03 I did the same for these types TYPE_KNATIVE_SERVICE,
        TYPE_EVENT_PUB_SUB,
        TYPE_EVENT_SOURCE,
        TYPE_EVENT_SOURCE_KAFKA